frontend/packages/knative-plugin/src/components/overview/KSRoutesOverviewListItem.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink, ExternalLink } from '@console/internal/components/utils';
+import { RouteModel } from '../../models';
+
+export type KSRoutesOverviewListItemProps = {
+  ksroute: K8sResourceKind;
+};
+
+const KSRoutesOverviewListItem: React.FC<KSRoutesOverviewListItemProps> = ({ ksroute }) => {
+  const {
+    metadata: { name, namespace },
+    status: { url, uid },
+  } = ksroute;
+  return (
+    <li key={uid} className="list-group-item">
```

Comment:
if you create the re-useable component mentioned above then you won't need to pass the key here